values.yaml

```diff
@@ -700,6 +703,9 @@ redis:
     image:
       repository: goharbor/redis-photon
       tag: dev
+    initContainerImage:                                                         
```

Comment:
You don't need a different init container image. You should simply reuse the `harbor-redis` container image. This way, you also have the redis username and group name, which means you can reference them directly instead of using the hardcoded UID and GID values.